table/tables/tables.go

```
@@ -723,6 +753,49 @@ func GetColDefaultValue(ctx context.Context, col *model.ColumnInfo) (interface{}
 	return col.DefaultValue, true, nil
 }
 
+// EncodeRecordKey encodes the string value to a byte slice.
+func EncodeRecordKey(tablePrefix string, h int64, columnID int64) []byte {
+	var (
+		buf []byte
+		err error
+	)
+
+	if columnID == 0 { // Ignore columnID.
+		buf, err = kv.EncodeValue(tablePrefix, h)
+	} else {
+		buf, err = kv.EncodeValue(tablePrefix, h, columnID)
+	}
+	if err != nil {
+		log.Fatal("should never happend")
```

Seems we should return an error instead of log.Fatal, maybe in next PR.